vars/main.yml

```diff
@@ -4,4 +4,12 @@ nrinfragent_os_version: "{{ ansible_lsb.major_release }}"
 nrinfragent_os_codename: "{{ ansible_lsb.codename }}"
 nrinfragent_distribution: "{{ ansible_distribution|lower }}"
 nrinfragent_architecture: "{{ {'x86_64': 'amd64', 'i386': '386'}[ansible_architecture] | default(ansible_architecture) }}"
-nrinfragent_config_path: "{{ nrinfragent_config_file | default((ansible_os_family == 'windows') | ternary('%ProgramFiles%\New Relic\newrelic-infra\newrelic-infra.yml', '/etc/newrelic-infra.yml'), true) }}"
\ No newline at end of file
+
+nrinfragent_config_path: >-
```

Comment:
It seems that this solution is not taking into account `nrinfragent_config_file` . It should get the value from `nrinfragent_config_file` and if empty, default to OS based one. 

Can you take this into account?

Thanks!